forms/FormField.php

```diff
@@ -138,6 +138,8 @@ public static function name_to_label($fieldName) {
 	 * @return string
 	 */
 	public static function create_tag($tag, $attributes, $content = null) {
+		Requirements::javascript(FRAMEWORK_DIR . '/javascript/FieldToggleDescription.js');
```

Comment:
This should be in the FromField() function, which is where all other formfields require their javascript.

It should also only be included if necessary. If you don't want to include this check, then you can put it in one of the existing framework/admin/javascript files; likely LeftAndMain.FieldHelp.js, which already has similar field helpers.